spec/models/facility_spec.rb

```diff
@@ -191,4 +191,17 @@
   describe 'Behavior' do
     it_behaves_like 'a record that is deletable'
   end
+
+  describe '#cohort_analytics' do
+    it 'should only consider registered hypertensive patients' do
```

Comment:
I suspect we'll need to add a bunch of contexts like this across the different touchpoints?

Reply:
We'll need a few in `MyFacilities`, the Cohort ones are covered in this PR. The Dashboard ones are covered in https://github.com/simpledotorg/simple-server/pull/897.